drivers/eks/eks_driver.go

```diff
@@ -211,6 +252,11 @@ func (state *state) validate() error {
 		return fmt.Errorf("client secret is required")
 	}
 
+	amiForRegion, ok := amiForRegionAndVersion[state.KubernetesVersion]
```

Comment:
I pulled in this commit into Rancher and tried to run it and I'm getting a failure right here.  I believe it is due to the fact that the UI is not sending a k8s version and we don't have a default.

Reply:
Yes, k8s version will be required for input when this comments is applied. I have a related PR in Rancher for the tests too.